system/api-server/src/file-upload/file-upload.controller.ts

```diff
@@ -58,7 +58,7 @@ export class ImageUploadController {
     @Res() response,
   ) {
     if (!file) {
-      throw new BadRequestException('invalid file provided');
+      throw new BadRequestException('Invalid file provided');
```

Comment:
Nitpick: for consistency's sake, use the same case for error messages

Reply:
Alright I will. Thanks!